py/server/deephaven/util/function_generated_table.py

```diff
@@ -0,0 +1,84 @@
+from typing import Callable, Union, List
+
+import jpy
+
+from deephaven import execution_context, DHError
+from deephaven.execution_context import ExecutionContext
+from deephaven.jcompat import j_lambda
+from deephaven.table import Table
+
+_J_Table = jpy.get_type("io.deephaven.engine.table.Table")
+_J_Supplier = jpy.get_type('java.util.function.Supplier')
+_J_FunctionGeneratedTableFactory = jpy.get_type("io.deephaven.engine.table.impl.util.FunctionGeneratedTableFactory")
+
+
+def create(table_generator: Callable[[], Table],
```

Comment:
Not sure if it should stand on its own. Maybe `table_factory.py` is a more fitting landing place.

Reply:
I haven't done a proper review, but I am very much against `util` packages.  They just become dumping grounds for garbage over time.  Also, this giant module name is not pythonic.  It is way too long.

Reply:
Moved it to `table_factory.py`